quiz.rb

```diff
@@ -2,19 +2,60 @@
 # Create a class called Dog. It should have, as attributes, name, breed, and age.
```

Comment:
Chris, sorry for taking so long to get your score back to you! I got interrupted during my grading this morning, and haven't had a chance to finish until just now.